.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build and verify
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
There are the [default](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request), so we can omit them.

Reply:
I see I forgot to push this.